src/js/components/awardv2/shared/additionalInfo/RecipientDetails.jsx

```diff
@@ -76,7 +90,7 @@ export default class RecipientDetails extends React.Component {
                                 className="accordion-table__row">
                                 <td>Recipient</td>
                                 <td>
-                                    {this.formatRecipientLink(this.props.data.internalId, this.props.data.name)}
+                                    {this.formatRecipientLink(this.props.data.internalId, this.props.data._name)}
```

Comment:
Should this be `name`? without the underscore?

Reply:
Good question - `name` maps to `Unknown` for nulls, whereas `_name` maps to the empty string for nulls. At the top of the page in the recipient section we use `name` to display Unknown, but here I'm using the falsy-ness of empty string to show `--` instead.

Reply:
wont is just display an empty string since `_name` will be an empty string if it is `null` and `formatRecipientLink` will just display the param `name` if there is no `internalId` and no `name`?

Reply:
nevermind, i thought formatRecipientLink was the same as Agency Recipient